gen/cycle.ml

```diff
@@ -741,16 +749,16 @@ let set_same_loc st n0 =
                    n.evt <- { n.evt with v = tr_value n.evt v; } ;
                    (* Writing Ord resets morello tag *)
                    let st = CoSt.set_co st CapaTag evt_null.ctag in
-                   let e,st = CoSt.set_cell st n.evt in
+                   let e,st = CoSt.set_cell st n ~rv:rv in
                    n.evt <- e ;
                    do_set_write_val st pte_val ns
                 | Tag|CapaTag|CapaSeal ->
-                   let st = CoSt.next_co st bank in
-                   let v = CoSt.get_co st bank in
-                   n.evt <- { n.evt with v = v; } ;
-                   let e,st = CoSt.set_tcell st n.evt in
-                   n.evt <- e ;
-                   do_set_write_val st pte_val ns
+                  let st = CoSt.next_co st bank in
```

Comment:
I don't think there is any meaningful change here, is there? It's only indentation.